UtiliKit.podspec

```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
 s.name             = 'UtiliKit'
-s.version          = '1.3.5'
+s.version          = '1.3.6'
```

Comment:
Let's not bump the version just yet. We'll do a separate release after this is merged into `master`.

Reply:
OK, I'll change that back.